ota.c

```diff
@@ -754,13 +777,16 @@ int   ota_get_file_ex(char * repo, char * version, char * file, int sector, byte
                     if (header) {
                         //printf("%s\n-------- %d\n", recv_buf, ret);
                         //parse Content-Length: xxxx
-                        location=strstr(recv_buf,"Content-Length: ");
+                        location=strstr_lc(recv_buf,"content-length:");
                         strchr(location,'\r')[0]=0;
-                        location+=16; //flush Content-Length: //
+			if (location[15] == ' ') {
+                            location++;
+                        }
+                        location+=15; //flush Content-Length: //
                         clength=atoi(location);
                         location[strlen(location)]='\r'; //in case the order changes
                         //parse Content-Range: bytes xxxx-yyyy/zzzz
-                        location=strstr(recv_buf,"Content-Range: bytes ");
+                        location=strstr_lc(recv_buf,"content-range: bytes ");
```

Comment:
this string should also be preceded by a \n and numbers adjusted accordingly

Reply:
Same.

Reply:
same ;-)